tests/findex/findex.test.ts

```diff
@@ -19,27 +18,101 @@ import {
 import { USERS } from "../data/users"
 import { expect, test } from "@jest/globals"
 import { createClient } from "redis"
+import { hexEncode } from "../../src/utils/utils"
+import { randomBytes } from "crypto"
+import sqlite3 = require("sqlite3")
```

Comment:
Can you use `import` here?

Reply:
yes `import..require` is a special form to accommodate CommonJS

Reply:
we have all those in `src/utils/utils.ts` for leb128 to fix as well

Reply:
another form is `import * as sqlite3 from "sqlite3"` ... but there are subtle differences....

From the mouth of the boss: https://stackoverflow.com/a/35706271/1263942

Reply:
I think `import sqlite3 from 'sqlite3'` was working on my machine but maybe I'm confused with something else…
Concerning leb128, it's fixed on my branch wasm